clusterloader2/pkg/prometheus/manifests/dashboards/master-dashboard.dashboard.py

```diff
@@ -486,6 +497,7 @@ def api_call_latency(title, verb, scope, threshold):
     refresh="",
     rows=[
         d.Row(title="Clusterloader", panels=CLUSTERLOADER_PANELS),
```

Comment:
At the moment those graphs have little to do with clusterloader. Can we just rename it to `API call latency` or sth similar?

Reply:
Sure, make sense. Renamed.